.github/workflows/test.yaml

```diff
@@ -30,3 +30,11 @@ jobs:
       branch: ${{ inputs.branch }}
       date: ${{ inputs.date }}
       sha: ${{ inputs.sha }}
+  java-tests:
```

Comment:
We do run Java tests in nightlies, so this can stay, but it needs to use a custom job.

Reply:
Added custom job